README.md

```diff
@@ -40,18 +41,21 @@ This role requires root access, so either configure it in your inventory files,
     - role: aalaesar.install_nextcloud
 ```
 
+## Dependencies
+
+- [geerlingguy.php-versions](https://github.com/geerlingguy/ansible-role-php-versions)
+
```

Comment:
Removing the dependency from the Meta file made me think about dependency installations.
Because I don't think all user may now, can we add
```suggestion

You can install it by running either `ansible-galaxy role install geerlingguy.php-versions`
or  `ansible-galaxy role install -r ./requirements.yml` when working in this role directory
```
as I think installing this role with galaxy will not pull automatically  the required role from the requirements.yml file